views/Main.tsx

```diff
@@ -14,13 +15,18 @@ interface Props {
 
 class Main extends Component<Props> {
     async componentDidMount() {
-        this.props.toggleView("meets")
 
         const res = await axios.post(`${getApiUrl()}/graphql`, {
             query: `
             query { Users {
+                id
                 first_name
                 last_name
+                email
+                phone_number
+                password_hash
```

Comment:
Remove this line. This is no longer being sent to the front-end and this will throw in error.